config/install/hm_newsletter.settings.yml

```diff
@@ -1,3 +1,6 @@
 hm_environment: 'production'
 hm_client_id: ''
-hm_imprint_text: ''
\ No newline at end of file
+hm_imprint_text: ''
+hm_displayed_agreements:
+  - 'datenschutzeinwilligung'
+  - 'anspracheerlaubnis'
```

Comment:
Add an update hook to prefill this config with the prviewsly hard coded agreement